packages/block/examples/6110Requests.ts

```diff
@@ -21,7 +21,7 @@ const main = async () => {
     signature: randomBytes(96),
     index: bytesToBigInt(randomBytes(8)),
   }
-  const request = DepositRequest.fromRequestData(depositRequestData) as CLRequest<CLRequestType>
+  const request = createDepositRequest(depositRequestData) as CLRequest<CLRequestType>
```

Comment:
We should investigate why this was not caught by the CI (I very much think we have a GH action testing the example running?)! 🤔